packages/spor-line-tag-react-native/src/LineIcon.tsx

```diff
@@ -0,0 +1,214 @@
+import React from "react";
+import { Box } from "@vygruppen/spor-layout-react-native";
+import {
+  AltTransportFill18Icon,
+  AltTransportFill24Icon,
+  AltTransportFill30Icon,
+  BusFill18Icon,
+  BusFill24Icon,
+  BusFill30Icon,
+  ExpressBusFill18Icon,
+  ExpressBusFill24Icon,
+  ExpressBusFill30Icon,
+  FerryFill18Icon,
+  FerryFill24Icon,
+  FerryFill30Icon,
+  SubwayFill18Icon,
+  SubwayFill24Icon,
+  SubwayFill30Icon,
+  TrainFill18Icon,
+  TrainFill24Icon,
+  TrainFill30Icon,
+  TramFill18Icon,
+  TramFill24Icon,
+  TramFill30Icon,
+  WalkFill18Icon,
+  WalkFill24Icon,
+  WalkFill30Icon,
+} from "@vygruppen/spor-icon-react-native";
+import {
+  BackgroundColorProps,
+  composeRestyleFunctions,
+  createVariant,
+  SpacingProps,
+  spacingShorthand,
+  SpacingShorthandProps,
+  useRestyle,
+  VariantProps,
+} from "@shopify/restyle";
+import { Theme } from "@vygruppen/spor-theme-react-native";
+import { StyleSheet } from "react-native";
+
+type Variant = VariantProps<Theme, "lineIconVariants", "variant">;
+const variant = createVariant({
+  themeKey: "lineIconVariants",
+});
+
+type RestyleProps = SpacingProps<Theme> &
+  SpacingShorthandProps<Theme> &
+  BackgroundColorProps<Theme> &
+  Variant;
+
+const restyleFunctions = composeRestyleFunctions<Theme, RestyleProps>([
+  spacingShorthand,
+  variant,
+]);
+
+type LineIconVariantProps =
+  | "local-train"
+  | "region-train"
+  | "region-express-train"
+  | "long-distance-train"
+  | "airport-express-train"
+  | "vy-bus"
+  | "local-bus"
+  | "ferry"
+  | "subway"
+  | "tram"
+  | "alt-transport"
+  | "walk";
+
+type LineIconSizeProps = "sm" | "md" | "lg";
+type LineIconTypeProps = "travel" | "info";
+
+type LineIconProps = Exclude<RestyleProps, "variant"> & {
+  variant: LineIconVariantProps;
+  size: LineIconSizeProps;
+  travelOrInfo: LineIconTypeProps;
```

Comment:
Kanskje rename denne til `lineIconType` eller noe slikt? Da binder vi oss ikke til to typer, men kan ekspandere med flere i fremtiden om man vil.